README.md

```diff
@@ -33,4 +33,5 @@ This repository contains my personal (primarily in Java save for some exceptions
 
 # Contributors
 Andre <...>
+Kai Ting <...>
```

Comment:
Hi Kaiting! i placed that <..> as a placeholder for any links you would like to put (eg to your own github acc)

feel free to remove it if u dw to put yet.